protobuf-definitions/make.sh

```diff
@@ -21,8 +21,8 @@ mkdir -p $DST_DIR_P/$PYTHON_PACKAGE
 
 # generate proto objects in python and C#
 
-protoc --proto_path=proto --csharp_out=$DST_DIR_C $SRC_DIR/*.proto
-protoc --proto_path=proto --python_out=$DST_DIR_P --mypy_out=$DST_DIR_P $SRC_DIR/*.proto
+$COMPILER/protoc --proto_path=proto --csharp_out=$DST_DIR_C $SRC_DIR/*.proto
```

Comment:
Should we update the comments above? Anything in https://github.com/Unity-Technologies/ml-agents/blob/develop/protobuf-definitions/README.md that can be cleaned up?

Reply:
Yes, now that the protobufs have been regenerated, I'm going to update the docs.